packages/desktop-gui/cypress/integration/project_nav_spec.js

```diff
@@ -122,20 +122,6 @@ describe('Project Nav', function () {
           .find('.dropdown-menu').first()
           // one is showing in the selection, so won't be in the list
           .find('.browser-icon')
-          .each(function ($i, i) {
-            const dropdownBrowsers = Cypress._.filter(this.config.browsers, (b) => {
-              // Chrome is shown in selection, so skip it
-              return b.displayName !== 'Chrome'
-            })
-
-            let family = dropdownBrowsers[i].family
-
-            family = family === 'electron' ? 'chrome' : family
-
-            // first one is shown in selection, so skip first
-            cy.wrap($i).should('have.class',
-              `fa-${family}`)
-          })
```

Comment:
This test needs to be added back and updated to ensure all the svg icons are displaying within the browser list.

Reply:
Sorry, maybe I have not been clear. The test needs to iterate through `this.config.browsers` (like it did before) for each icon and ensure that the class matches the `this.config.browsers.name`.

 We do not need a test that '.browser-icons' is a class, but that the exact icon is being displayed for the browser that should be shown in the list. So for each item in the dropdown it should have class `.chrome`, `.chromium`, `.canary`, etc. in order, removing the selected browser.

Reply:
Hi again @jennifer-shehane , first of all i'm sorry for the misunderstanding, i hope that this time i have understood well not to spend your time more hahaha, i'm trying to give my best and all your recommendations are helping me :D
If there is any other improvement that you see convenient to do just tell me :)